cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -194,7 +195,7 @@ public Provisioner provisioner() {
    * @return A new auto closeable semaphore for the cluster model generation.
    */
   public LoadMonitor.AutoCloseableSemaphore acquireForModelGeneration(OperationProgress operationProgress)
-      throws InterruptedException {
+          throws InterruptedException {
```

Comment:
Please revert this if unnecessary: if it doesn't violate checkstyle rules, then preferably we shouldn't do stylistic changes as the change itself is big enough as it is.